openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/AbstractStream.java

```diff
@@ -48,8 +49,15 @@ public void release() {
 
 	public abstract void release(boolean remove);
 
-	public static WebRtcEndpoint createWebRtcEndpoint(MediaPipeline pipeline, Boolean recv) {
+	public static WebRtcEndpoint createWebRtcEndpoint(MediaPipeline pipeline, Boolean recv,
+			String certificateType) {
 		WebRtcEndpoint.Builder builder = new WebRtcEndpoint.Builder(pipeline);
+		// See https://doc-kurento.readthedocs.io/en/latest/features/security.html#media-plane-security-dtls
+		if (CertificateKeyType.RSA.name().equals(certificateType)) {
```

Comment:
I would do `if` or `switch` on `kHandler.getCertificateType()` here

Reply:
That would mean you need to pass in the entire KHandler into the method ?
AbstractStream is not a bean. It requires parameters to be passed in.

Reply:
yep
this should be 

I would do if or switch on passed `CertificateKeyType` here :)